content/enterprise_features/sakuli-dashboard-configuration.md

```diff
@@ -0,0 +1,166 @@
+             }
+         },
+         "url":"https://sakuli.io/docs"
+      }
+   ]
+}
+{{</highlight>}}
+
+Let us examine the different sections:
+
+1. The `index` defines the order for the displays on the Sakuli dashboard. 
+2. The `messages` property is optional and can define the title using the `description` property and tool tip for the display using the `infoText`
+property. The content can be displayed in german or english using the appropriate section.
+3. The Sakuli dashboard embeds the `url` in an iFrame. 
+4. The `actionIdentifier` property is optional and references to an action defined within the [ACTION_CONFIG](#action_config).
+
+### ACTION_CONFIG {#action_config}
+
+With the `ACTION_CONFIG` you can configure the actions that can be triggered by users and cronjobs.
+Here is a sample `ACTION_CONFIG` for the Sakuli dashboard. 
+
+{{<highlight javascript>}}
+{
+   "actions":[
+      {
+         "actionIdentifier":"your_action_id_123",    //1
+         "action": {                                 //2
+            "metadata": {
+              "labels": {
+                "app": "sakuli"
+              },
+              "name":"sakuli"
+            },
+            "spec": {
+              "containers": [
+                {
+                  "name": "sakuli",
+                  "image": "taconsol/sakuli:latest",
+                  "env": [
+                    {
+                      "name": "VNC_VIEW_ONLY",
+                      "value": "true"
+                    },
+                    {
+                      "name": "SAKULI_ENCRYPTION_KEY",
+                      "valueFrom": {
+                        "secretKeyRef": {
+                          "name": "sakuli-encryption-key",
+                          "key": "key"
+                        }
+                      }
+                    }
+                  ]
+                }
+              ],
+              "restartPolicy": "Never"
+            }
+         }
+      }
+   ]
+}
+{{</highlight>}}
+
+1. Action identifier that is referenced inside `DASHBOARD_CONFIG`.
+2. Applies the Kubernetes/Openshift template on the cluster. In this case, the action represents a Kubernetes pod template
+to start a Sakuli check.
+
+### CLUSTER_CONFIG {#cluster_config}
+
+With the `CLUSTER_CONFIG` you can enable the access to an existing cluster where you plan to execute your actions.
+{{<highlight javascript>}}
+{
+   "cluster":{                                              //1
+      "name":"sakuli/examplecluster-com:443/developer",     //2           
+      "server":"http://examplecluster.com:443"              //3
+   },
+   "user":{                                                 //4
+      "name":"developer",         
+      "token":"<login-token>"     
+   },
+   "namespace":"sakuli"                                     //5
+}
+{{</highlight>}}
+
+1. Cluster for hosting dashboard.
+2. Name of Cluster: `<namespace>/<cluster-address>:<port>/<user>`
```

Comment:
In theory the cluster name is just a label to be set. It doesn't have to match the pattern you described as far as I understand